Integrations/Gmail/Gmail.py

```diff
@@ -905,17 +912,27 @@ def get_user_tokens(user_id):
 
 
 def search_all_mailboxes():
-    command_args = {
-        'maxResults': 100,
-        'domain': ADMIN_EMAIL.split('@')[1],  # type: ignore
-    }
-
+    next_page_token = None
     service = get_service('admin', 'directory_v1')
-    result = service.users().list(**command_args).execute()
+    while True:
+        command_args = {
+            'maxResults': 100,
+            'domain': ADMIN_EMAIL.split('@')[1],  # type: ignore
+            'pageToken': next_page_token
+        }
+
+        result = service.users().list(**command_args).execute()
+        next_page_token = result.get('nextPageToken')
+
+        entries = [search_command(user['primaryEmail']) for user in result['users']]
+
+        # if these are the final result push - return them
+        if next_page_token is None:
+            entries.append("Search completed")
```

Comment:
@kirbles19 
while the command runs, it will output entries to the war-room.
this message states that the command finally finished.
can you feedback on this message as well?

Reply:
@kirbles19 - just making sure, that you approved this as well when approving the docs.

Reply:
talked to @kirbles19 - this message is fine.